db/sql/blocks/aggregateBlocksReward.sql

```diff
@@ -0,0 +1,41 @@
+/*
+  DESCRIPTION: ?
+
+  PARAMETERS: ?
```

Comment:
This Query accepts some params. We should document these here. Applies to every SQL file.

Reply:
Yes, but again, I wouldn't be able to do it within this PR. There should be a separate, SQL documentation PR for that.